.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+dist: trusty
+language: python
+python:
+  - "3.5"
+
+before_install:
+  - sudo apt-get update -q || true
```

Comment:
We don't need sudo, do we? https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise#L451